Backpack-SwiftUI/Spacing/Classes/Generated/Spacing.swift

```diff
@@ -0,0 +1,94 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+import SwiftUI
+
+public struct Spacing {
```

Comment:
Would it be a bad idea to call this BPKSpacing? (To stay on brand).
Also wondering if we could have done with an extension on CGFloat, that way we wont need to extend any of the initializers, which would reduce toil in the future

Reply:
This gives us more control on how to apply the spacing to the components.
For SwiftUI native components, we need to create the extensions that you see in this file. But for our BackpackSwiftUI components, it gives us full control of the component and blocks misusage by design.
For example, if our components receive a cgfloat for setting a padding, this makes it possible to set a random number that's not been defined by our guidelines, which goes agains our goal of standardising the product

Reply:
keen to discuss further!

Reply:
There is a case for files that will have this problem, like Color for example, there we wouldn't have a choice but to name it BPKColor. Stablishing BPK as a prefix may be the way to go for consistency across backpack supported tokens/components

Reply:
fyi, I adopted the BPK prefix suggestion!